src/Nancy.Testing.Tests/BrowserFixture.cs

```diff
@@ -538,7 +538,7 @@ public void Should_override_default_user_agent_when_explicitly_defined()
 
             // Then
             header.ShouldEqual(expectedHeaderValue);
-        }
+        }     

```

Comment:
I am actually running that, but I was not on the machine I used to send the PR :P